docs/stdlib/math.rst

```diff
@@ -124,7 +123,7 @@ Math
 
     :index: average avg
 
-    Return the arithmetic mean of the input set.
+    Returns the arithmetic mean (i.e., average) of the input set.
```

Comment:
```suggestion
    Returns the arithmetic mean of the input set.
```
There are different notions of "average": mean, median and mode. This function is specifically the mean. If additional clarification referring to "average" is neccessary, it should be in the main body of the description.